crates/bitcoind_rpc/tests/test_emitter.rs

```diff
@@ -731,3 +744,109 @@ fn no_agreement_point() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn test_expect_tx_evicted() -> anyhow::Result<()> {
```

Comment:
Would you be able to add a brief summary at the top of the test (e.g. as a doc comment) explaining what the test is validating?

Without a summary, it’s a bit hard to grasp the intent of the test without reading through every line and trying to hold the context in mind. A short explanation would really help with readability, especially for larger tests like this one.